ui/app/rif/components/luminoNetworkChannels.js

```diff
@@ -97,21 +97,22 @@ class LuminoNetworkChannels extends Component {
               classes={classes}
             />
           </div>
-      }
-      {
-        data.length === 0 &&
-        <div>
-          <span className={classes.title}>Lumino Channels</span>
-          <span className={classes.notFound}>No channels found</span>
-        </div>
-      }
+        }
```

Comment:
Is this 2 or 4 identation spaces? be careful with that, specially if you use visua studio as an IDE.

Reply:
I am using Webstorm, so it is 2 identation spaces. (I believe)